File: angular/src/app/service/model/chart-settings/chart-detail-settings/chart-detail-selection.dto.ts
import { SelectionNameIdDto, SelectionKeyValueDto } from './selection-base-info.dto'

export class ChartDetailSelectionDto {
    branches: SelectionNameIdDto[] 
    jobPositions: SelectionNameIdDto[] 
    levels: SelectionNameIdDto[] 
    teams: SelectionNameIdDto[] 
    payslipDetailTypes: SelectionKeyValueDto[] 
    userTypes: SelectionKeyValueDto[] 
    workingStatuses: SelectionKeyValueDto[] 
    gender: SelectionKeyValueDto[]

}

File: aspnet-core/src/HRMv2.Core/Manager/Categories/Charts/ChartDetails/Dto/ChartDetailSelectionDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static HRMv2.Constants.Enum.HRMEnum;

namespace HRMv2.Manager.Categories.Charts.ChartDetails.Dto
{
    public class ChartDetailSelectionDto
    {
        public List<KeyValueDto> JobPositions { get; set; }
        public List<KeyValueDto> Branches { get; set; }
        public List<KeyValueDto> Levels { get; set; }
        public List<KeyValueDto> Teams { get; set; }
        public List<KeyValueDto> UserTypes { get; set; }
        public List<KeyValueDto> PayslipDetailTypes { get; set; }
        public List<KeyValueDto> Gender { get; set; }
        public List<KeyValueDto> WorkingStatuses { get; set; }

    }
}
